docs/content/paket-pack.md

```diff
@@ -0,0 +1,11 @@
+# paket pack
+Packs all paket.template files within this repository
+
+    [lang=batchfile]
+    $ paket pack output outputDirectory buildconfig Debug
+
+Options:
+
+  `output`: Output directory to put nupkgs
+
+  `buildconfig`: Optionally specify build configuration that should be packaged (defaults to Release).
```

Comment:
yes. It should also go into helptexts. This will allow us to call `paket pack --help` and get the text in the command line